Ship the gettext translations as /usr/share/locales/*/gnutls13.mo instead of
/usr/share/locales/*/gnutls.mo to not break co-installability of 
multiple libgnutls packages.

--- gnutls13-1.6.1/po/Makevars.orig	2006-03-03 16:16:16.000000000 +0000
+++ gnutls13-1.6.1/po/Makevars	2007-02-11 10:22:01.000000000 +0000
@@ -1,7 +1,7 @@
 # Makefile variables for PO directory in any package using GNU gettext.
 
 # Usually the message domain is the same as the package name.
-DOMAIN = $(PACKAGE)
+DOMAIN = $(PACKAGE)26
 
 # These two variables depend on the location of this directory.
 subdir = po
